lib/agent0/agent0/utilities/docker.py

```diff
@@ -0,0 +1,60 @@
+"""Utilities associated to manipulating Docker."""
+import subprocess
+import logging
+import os
+import time
+from pathlib import Path
+
+
+def check_docker(restart: bool = False) -> None:
+    """Check whether docker is running to your liking.
+
+    Parameters
+    ----------
+    restart: bool
+        Restart docker even if it is running.
+    """
+    home_infra = Path(os.path.expanduser("~")) / "code" / "infra"
```

Comment:
I don't think we should be hard-coding this path here if you want this utility to be useful for other people

Reply:
maybe I should pass it in as an optional local_path

Reply:
added optional local path parameter in ea27a0edcbb15a0c5d5f6b03abc8d562274985bc